### PR TITLE
[EAM-1675] New component to visualize the documents and instructions attached to a TaskPlan

### PR DESCRIPTION
**Pull request by purquijo**
New component to visualize the documents and instructions attached to a TaskPlan. In order to do it so a new WS class has been created that points to the cern-eam-services. This can be useful in a future.

The documents are displayed inside an iframe. The UI of this iframe is browser dependant. If in a future we want to unify this, a custom solution should be implemented.

### CODE REVIEW

**Comment by purquijo** [comment]
@ricardo-j-almeida @LukaszPater should we disable the component for open-source?

**Comment by ricardo-j-almeida** [comment]
> @ricardo-j-almeida @LukaszPater should we disable the component for open-source?

If you mean disable as in it shouldn't render at all then yes

File: src/ui/components/checklists/Checklists.js [hunk 1]
```diff
@@ -717,6 +738,12 @@ class Checklists extends Component {
                                 {this.props.bottomSlot}
                             </BlockUi>
                             <Dialog open={this.state.createFollowUpActivity !== null}>{dialog}</Dialog> 
+                            {isCernMode && (
+                                <>
+                                    <iframe src="https://testedms.cern.ch/ui/SsoLoginServlet" style={{ width: 0, height: 0, display: 'none' }}></iframe>
+                                    <iframe src="https:/edms.cern.ch/ui/SsoLoginServlet" style={{ width: 0, height: 0, display: 'none' }}></iframe>
```
Comment (ricardo-j-almeida) [inline src/ui/components/checklists/Checklists.js:744]: We should only initialize the iframe for the correct environment

Comment (purquijo) [reply src/ui/components/checklists/Checklists.js]: @ricardo-j-almeida done. What do we do with the dist conflicts?

Comment (purquijo) [reply src/ui/components/checklists/Checklists.js]: The install parameter EL_EDMSS must be created in prod with the value: https://edms.cern.ch/ui/SsoLoginServlet

**Comment by ricardo-j-almeida** [comment]
Please don't include the dist changes, as the build/deployment script handles that.

**Comment by purquijo** [comment]
@ricardo-j-almeida done

**Pull request merged**